src/test/scala/com/jakainas/functions/functionsTest.scala

```diff
@@ -42,6 +42,23 @@ class functionsTest extends SparkTest {
     an [NullPointerException] should be thrownBy plusDays(null, 5)
   }
 
+  test("dateCols: numeric input"){
```

Comment:
test titles should ideally be a bit more descriptive, something along the lines of what's being given and what should be returned e.g. 
`dateCols should return a properly formatted String date given valid numeric inputs`